_i18n/en.yml

```diff
@@ -1267,144 +1284,184 @@ playbook_page:
 
-    ### For more information
+        Protecting your information is our priority. You can read below for a more detailed explanation of our privacy and security practices.
 
-    We are happy to answer questions about our security and privacy practices.
-    For more information, please visit [Help](site.baseurl/help)
-    or [contact us](site.baseurl/contact).
-- section: Messaging terms and conditions
-  anchor: our-messaging-practices
-  content: |-
-    Login.gov uses SMS text messaging to help verify your identity when you create
-    a new account.  SMS text messages, or automated voice calls, may also be used if
-    you choose your telephone number as one of your authentication methods.
-
-    ### SMS Terms and Conditions
-    - Login.gov provides an authentication service and will not use text or
-      voice messaging for any other purpose.
-    - Your carriers messaging and data rates may apply.
-    - Messages will only be sent as part of setting up and signing into your
-      login.gov account.
-    -	You may [contact customer care here](site.baseurl/contact) or by email at <hello@login.gov>.
-    -	To opt out of receiving text messages from login.gov reply with the message _STOP_
-      to the unwanted text message.
-    -	See [Our Privacy Practices](#our-privacy-practices) for privacy policy information.
-    -	Carriers are not liable for delayed or undelivered messages.
-policy_page:
-  content:
-    p_1: |-
-      Your privacy is very important to us. We’re providing you with
-      our privacy policy so you are aware of what information we collect,
-      why we collect it, and what we do with it. login.gov does not collect
-      your email address or phone number unless you choose to provide it.
-      We do collect other limited information automatically from visitors
-      who read or browse information from our site. We do this to better
-      understand how the site is being used and how we can make it more
-      helpful.
-    p_2: |-
-      If you create an account, login.gov collects personally identifiable
-      information (PII) from you, including your email address and phone number.
-      We collect your email address and with your consent share it with each
-      federal agency (“partner agency”) at which you are seeking to access
-      information and services. We collect your phone number in order to enable
-      two-factor authentication as a security measure for your login.gov
-      account. If you’re accessing a government application that requires
-      identity proofing, login.gov will collect, use, maintain, and share
-      additional PII for the sole purpose of identity proofing.
+        * [How does it work?](site.baseurl/policy/how-does-it-work/)
```

Comment:
I think that the `site.baseurl` here is messing up the external link checker, because it sees a domain and checks the link out in the real internet --- can we use an absolute URL here?

Reply:
If I do that, then on the local build, the link, when viewed from a translated page, will point to:
`/policy/how-does-it-work/` instead of `/es/policy/how-does-it-work`

Reply:
What if we add in a replacement for `site.lang` then?

Reply:
But putting the domain in the URL is causing a problem - still trying to think of a nice solution.

Reply:
Revisiting `site.lang`...

Reply:
Simply converting the `site.baseurl` string above into the real one was, as you said, adding a domain which would fail the external link tests.  I've added a custom `lang_path` filter (for some brevity) that replaces this with the proper language prefix (or "" if english).

For example:
```
{% assign lpath = site.lang | lang_path %}
{{ site.translations[site.lang]["policies"]["sections"][page.index]["content"] | replace: "site.baseurl", lpath | markdownify }}
```
Local testing is going well...